# Exception Handling

1. Wrap the command that could throw an error inside of a `try` / `catch` block
2. In the `catch` block, `return` your preferred result

```javascript
try {
  return await this.find(locator).isDisplayed()
} catch (error) {
  return false
}
```

For more info see:

+ [the MDN article on `try...catch`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch)


